migtests/tests/pg/datatypes/validate

```diff
@@ -46,9 +46,22 @@ EXPECTED_SUM_OF_COLUMN = {
 EXPECTED_ENUM_VALUES = ['mon', 'tue', 'wed', 'thu', 'fri', 'sat', 'sun']
 EXPECTED_BYTE_TYPE_VALUES = ['3031303130', '2d61626364', '34343538']
 
+EXPECTED_COLOCATION_OF_TABLES = {
+	'num_types': 't',   
+	'datatypes1': 't',  
+	'datetime_type': 't', 
+	'datatypes2': 't',
+	'datetime_type2': 't', 
+	'null_and_default' : 't',
+	'decimal_types': 't',
+}
+
 def migration_completed_checks_yb():
 	print("Running tests on YB")
 	yb.run_checks(migration_completed_checks)
+	# Commenting the validation for now due to the large sized source database requirements.
```

Comment:
Didn't understand why we need to comment this?
Yes, all tables will be determined to be colocated, but that's what you're asserting as well, right?

Reply:
Yes I was confused as to what to do here. 
Removing the comment.